app/src/common/shared/org/mozilla/vrbrowser/ui/SettingsStore.java

```diff
@@ -73,4 +73,25 @@ public void setTelemetryEnabled(boolean isEnabled) {
         TelemetryHolder.get().getConfiguration().setCollectionEnabled(isEnabled);
     }
 
+
+    public void setCurrentSeesionId(int sessionId) {
+        SharedPreferences.Editor editor = mPrefs.edit();
+        editor.putInt(mContext.getString(R.string.settings_key_session_id), sessionId);
+        editor.commit();
+    }
+
+    public int getCurrentSessionId() {
+        return mPrefs.getInt(mContext.getString(R.string.settings_key_session_id), SessionStore.NO_SESSION_ID);
+    }
+
+    public void setPreviousSeesionId(int sessionId) {
+        SharedPreferences.Editor editor = mPrefs.edit();
+        editor.putInt(mContext.getString(R.string.settings_key_previous_session_id), sessionId);
+        editor.commit();
+    }
+
+    public int getPreviousSessionId() {
```

Comment:
Would it make sense to name the methods: getCurrentSessionId() and getCurrentPrivateSessionId()
I think it's clearer to call them that way if I have understand the usage correctly.

Reply:
mmm you'd also need a extra variable to know if the current is public/private. So it's ok